Docs/source/install/hpc/ookami.rst

```diff
@@ -0,0 +1,126 @@
+.. _building-ookami:
+
+Ookami (Stony Brook)
+====================
+
+The `Ookami cluster <https://www.stonybrook.edu/ookami/>`__ is located at Stony Brook University.
+
+If you are new to this system, please see the following resources:
+
+* `Ookami documentation <https://www.stonybrook.edu/commcms/ookami/support/index_links_and_docs.php>`__
+* Batch system: *TODO*
+* Production directories *TODO*:
+
+  * A high-performance Lustre filesystem provides about 0.8 PByte storage.
+  * ...
+
+We use Ookami as a development cluster for `A64FX <https://www.arm.com/blogs/blueprint/fujitsu-a64fx-arm>`__,
+The cluster also provides a few extra nodes, e.g. two ``Thunder X2`` (ARM) nodes.
+
+
+
+Installation
+------------
+
+Use the following commands to download the WarpX source code and switch to the correct branch:
+
+.. code-block:: bash
+
+   git clone https://github.com/ECP-WarpX/WarpX.git $HOME/src/warpx
+
+We use the following modules and environments on the system (``$HOME/warpx_gcc10.profile``).
+
+.. code-block:: bash
+
+   # please set your project account (not relevant yet)
+   #export proj=<yourProject>
+
+   # required dependencies
+   module load cmake/3.19.0
+   module load gcc/10.3.0
+   module load openmpi/gcc10/4.1.0
+
+   # optional: faster builds (not available yet)
+   #module load ccache
+   #module load ninja
+
+   # optional: for PSATD support (not available yet)
+   #module load fftw
+
+   # optional: for QED lookup table generation support (not available yet)
+   #module load boost
+
+   # optional: for openPMD support
+   #module load adios2  # not available yet
+   #module load hdf5    # only serial
+
+   # compiler environment hints
+   export CC=$(which gcc)
+   export CXX=$(which g++)
+   export FC=$(which gfortran)
+   export CXXFLAGS="-mcpu=a64fx"
+
+
+We recommend to store the above lines in a file, such as ``$HOME/warpx_gcc10.profile``, and load it into your shell after a login:
+
+.. code-block:: bash
+
+   source $HOME/warpx_gcc10.profile
+
+
+Then, ``cd`` into the directory ``$HOME/src/warpx`` and use the following commands to compile:
+
+.. code-block:: bash
+
+   cd $HOME/src/warpx
+   rm -rf build
+
+   cmake -S . -B build -DWarpX_COMPUTE=OMP -DWarpX_OPENPMD=ON -DAMReX_MPI_THREAD_MULTIPLE=FALSE
```

Comment:
```suggestion
   cmake -S . -B build -DWarpX_COMPUTE=OMP -DWarpX_OPENPMD=ON
```
I did a quick test and it seems to me that on Ookami this is not needed

Reply:
Oh, I tested this with MVAPICH but might not have re-tested with OpenMPI. Thanks for checking!